src/filtrering/filter-konstanter.ts

```diff
@@ -211,8 +211,6 @@ export const aktiviteter = {
     UTDANNINGAKTIVITET: 'Utdanning og kurs (for enslige forsørgere eller egenfinansiert)'
 };
 
-const veiledere = {};
```

Comment:
I am just curious why this was here...

Reply:
i may have added it a looong time ago to check something and forgot to remove it 😅